docs/src/main/asciidoc/amazon-lambda-http-guide.adoc

```diff
@@ -35,4 +35,108 @@ Quarkus project, you'll need to configure your application for deployment to Ama
 the scope of this document but you can find the full guide on
 https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-getting-started.html[Amazon's website].
 
-For the purposes of this guide, we'll be using RESTEasy and maven.
\ No newline at end of file
+For the purposes of this guide, we'll be using RESTEasy and maven.  In order to deploy your REST application on AWS,
+you'll need to add two dependencies:
+
+```xml
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-amazon-lambda-http</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-resteasy</artifactId>
```

Comment:
It seems that this dependency is required to function properly, so users may ask why it isn't a transitive dependency of the `quarkus-amazon-lambda-http` extension?

However below you explain that `It's likely there is already an HTTP dependency defined to serve your application`. Perhaps it would be nice to explain that here.

Reply:
The theory is that users can choose quarkus-resteasy or quarkus-vertx-web or quarkus-undertow or maybe already have.

Reply:
I was also not clear on what the guide was trying to say in the part that @gastaldi quoted. Perhaps it would be good to explain what choices users have and what each one means for lambda?